### ISSUE TITLE
Can I get the buffering duration of Sonos via Sonos API functions?

### ISSUE THREAD

**Issue by chaocharleswang** [jishi/node-sonos-http-api#593]
I use Sonos to play some online streaming video or audio.
Can I get the buffering duration of Sonos via Sonos API functions?

**Comment by jishi** [comment]
No, not to my knowledge. When you say video, how are you streaming that?
